ts/receiver/groups.ts

```diff
@@ -109,12 +111,15 @@ export async function preprocessGroupMessage(
       }
     }
     // send a session request for all the members we do not have a session with
-    window.libloki.api.sendSessionRequestsToMembers(group.members);
+    await window.libloki.api.sendSessionRequestsToMembers(group.members);
   } else if (newGroup) {
     // We have an unknown group, we should request info from the sender
-    window.textsecure.messaging.requestGroupInfo(conversationId, [
-      primarySource,
-    ]);
+    const requestInfo = {
+      timestamp: Date.now(),
+      groupId: conversationId,
+    };
+    const requestInfoMessage = new ClosedGroupRequestInfoMessage(requestInfo);
+    await getMessageQueue().sendToGroup(requestInfoMessage);
```

Comment:
This message needs to be sent to the `sender`

Reply:
fixed